frontend/styles/Login.module.css

```
@@ -0,0 +1,70 @@
+.login-form {
+  width: 500px;
+  height: auto;
+  margin: 5% auto;
```

This `5%` makes for an abnormally large gap on larger screens. It should be set to a static value instead, such as `32px`.
```suggestion
  margin: 32px auto;
```